modules/ROOT/pages/custom-policy-getting-started.adoc

```diff
@@ -52,25 +47,34 @@ One way to do so is by configuring Maven's `settings.xml` with the following sec
  </profiles>
 ----
 
-Once Maven is configured
+Once Maven is configured, follow the following steps:
 
-. Create a new directory where the custom policy project will live
-. Go to that directory in the command line
+. Create a new directory where the custom policy project will live.
+. Go to that directory in the command line.
 . Execute the following command:
-
++
 [source,console,linenums]
 ----
-mvn -Parchetype-repository archetype:generate -DarchetypeGroupId=org.mule.tools -DarchetypeArtifactId=api-gateway-custom-policy-archetype -DarchetypeVersion=1.0.0 -DgroupId=${orgId} -DartifactId=${policyName} -Dversion=1.0.0-SNAPSHOT
+mvn -Parchetype-repository archetype:generate \
+-DarchetypeGroupId=org.mule.tools \
+-DarchetypeArtifactId=api-gateway-custom-policy-archetype \
+-DarchetypeVersion=1.0.0 \
+-DgroupId=${orgId} \
+-DartifactId=${policyName} \
+-Dversion=1.0.0-SNAPSHOT \
+-Dpackage=mule-policy
 ----
-
++
 Replacing:
-
-* `${orgId}` with Anypoint Plaform Organization Id where the policy will be uploaded to.
++
+* `${orgId}` with Anypoint Platform Organization Id where the policy will be uploaded to.
```

Comment:
"with the Anypoint..." 
Also, in this case, I'd delete the final "to" for a smoother read.